easybuild/easyblocks/p/perl.py

```diff
@@ -139,7 +139,7 @@ def test_step(self):
             # specify locale to be used, to avoid that a handful of tests fail
             cmd = "export LC_ALL=C && %s" % cmd
 
-            run_cmd(cmd, log_all=False, log_ok=False, simple=False)
+            run_shell_cmd(cmd, fail_on_error=False)
```

Comment:
Why are we ignoring non-zero exit codes here? That seems wrong...

@bartoldeman I understand that you're just translating from `run_cmd` to `run_shell_cmd` without changing behavior here, but this looks like a bug we should fix, no?

Reply:
see #3170